docs/user-guide/ze-install.md

```diff
@@ -54,41 +55,69 @@ You can also watch the following videos to learn how to get started with Zowe Ex
 
 <iframe class="embed-responsive-item" id="youtubeplayer2" title="How to Work with Data Sets" type="text/html" width="100%" height="365" src="https://www.youtube.com/embed/X4oSHrI4oN4" frameborder="0" webkitallowfullscreen="true" mozallowfullscreen="true" allowfullscreen="true"> </iframe>
 
-## Configuration
+## Configuring Zowe Explorer
 
-Configure Zowe Explorer in the settings file of the extension. To access the extension settings, navigate to **Manage (the gear icon)**  > **Settings**, then select **Extensions** > **Zowe Explorer Settings**. For example, you can modify the following settings:
+Configure Zowe Explorer in the settings file of the extension.
 
-* **Data set creation settings:** You can change the default creation settings for various data set types.
+To access the extension settings, follow these steps:
 
-**Follow these steps:**
+1. Click the **Settings** icon at the bottom of the **Activity Bar**.
 
-1. Click the **Edit in settings.json** button under the Data Set, USS or JOBS settings that you want to edit.
-2. Edit the settings as needed.
-3. Save the settings.
+2. Select the **Settings** option.
+3. Open the **Extension** option listed in the **Commonly Used** menu.
+4. Select **Zowe Explorer** to access its settings.
+5. Scroll the list to find the setting that needs modification.
 
-<img src={require("../images/ze/ZE-Configuration.gif").default} alt="Configure Zowe settings"/>
+### Modifying creation settings for data sets, USS files, and jobs
 
-* **Set the Temporary Folder Location:** You can change the default folder location where temporary files are stored. 
+Follow these steps:
 
-    **Follow these steps:**
+1. In Zowe Explorer settings, scroll to a data set, USS file, or job setting type.
+2. Click the setting's corresponding **Edit in settings.json** link.
 
-   1. Click the **Edit in settings.json** button under the Data Set, USS or JOBS settings that you want to edit.
-   2. Modify the following definition:
+    This opens the `settings.json` file in an **Editor** tab. (The suggestions widget also opens if the functionality is enabled.)
 
-    ```json
-    "Zowe-Temp-Folder-Location": {
-        "folderPath": "/path/to/directory"
-      }
-    ```
+3. Edit the settings in the file as needed.
+4. Save the file to keep changes.
 
-  where **/path/to/directory** is the folder location that you specify.
-  
-  3. Save the settings.
+    ![Configure Zowe settings](../images/ze/ZE-Configuration2.gif)
+
+### Modifying temporary file location settings
+
+Change the default folder location where temporary files are stored with the following steps:
+
+   1. Navigate to Zowe Explorer settings.
+   2. Under the data set, USS, or jobs settings that you want to edit, click the **Edit in settings.json** link.
```

Comment:
You can also click on the top-right corner icon in the VS Code settings.
![image](https://user-images.githubusercontent.com/37381190/210865221-e201d7a1-f1b6-42fd-adfd-911bcf1ea28b.png)

However, navigating to the extension settings will try to open the `settings.json` file in context 😋